config/example-extras.cfg

```diff
@@ -583,6 +583,38 @@
 #a0_pin
 #   The pins connected to an uc1701 type lcd. These parameters must be
 #   provided when using an uc1701 display.
+#menu_root:
+#   Entry point for menu, root menuitem name. This parameter must be
+#   provided when using menu.
+#menu_timeout:
+#   Timeout for menu. Being inactive this amount of seconds will trigger
+#   menu exit or return to root menu when having autorun enabled [optional]
```

Comment:
Very minor, but at least for example-extras.cfg, the field descriptions should follow the same style as the rest of the config: [optional] should be replaced with "The default is xxx seconds"